test/linalg/diagonal.jl

```diff
@@ -116,6 +123,12 @@ for relty in (Float32, Float64, BigFloat), elty in (relty, Complex{relty})
                 @test Ac_mul_B!(copy(D), copy(b)) ≈ full(D)'*full(b)
             end
 
+            #a few missing mults
+            bd = Bidiagonal(D2)
+            @test A_mul_Bt(D,D2) ≈ full(D)*full(D2).'
```

Comment:
I wanted to make sure I would get this method. Is there a way to use `@which` to see what `(*)` will call?